buildshiprun/handler.go

```diff
@@ -298,6 +299,7 @@ func getEvent() (*sdk.Event, error) {
 	info.Image = os.Getenv("Http_Image")
 	info.SCM = os.Getenv("Http_Scm")
 	info.Private, _ = strconv.ParseBool(os.Getenv("Http_Private"))
+	info.RepoURL = os.Getenv("Http_Repo_url")
```

Comment:
Have you checked this is the exact value that is sent in? `Http_Repo_url` Casing looks a bit odd, please check. Passing in `Repo-URL` I get:

```
Http_Repo_Url=http://test
```

Reply:
Yeah I was too literal `Repo_URL` gives `Http_Repo_url` I will fix that up in a minute